src/libraries/System.Net.Http/src/System/Net/Http/Headers/HttpRequestHeaders.cs

```diff
@@ -19,8 +19,11 @@ public sealed class HttpRequestHeaders : HttpHeaders
 
         private object[]? _specialCollectionsSlots;
         private HttpGeneralHeaders? _generalHeaders;
+
+#if !TARGET_BROWSER
```

Comment:
Please revert the changes in this file.  If we want to throw a PNSE from the browser handler, we can do that in the browser handler in response to an Expect: 100-continue header being part of the request, or just let it fail or be ignored or whatever the browser will actually do with the header.

Reply:
Okay will let the browser deal with it, which is that they ignore it silently. I didn't have strong preference either way.